sklearn/ensemble/tests/test_forest.py

```diff
@@ -94,6 +97,8 @@
 FOREST_ESTIMATORS.update(FOREST_REGRESSORS)
 FOREST_ESTIMATORS.update(FOREST_TRANSFORMERS)
 
+FOREST_CLASSIFIERS_REGRESSORS = dict(FOREST_CLASSIFIERS, **FOREST_REGRESSORS)
```

Comment:
I got a bit surprise here. Apparently this is a cool hack (maybe not the best to have it in the code base since it working only with key string [cf.](http://treyhunner.com/2016/02/how-to-merge-dictionaries-in-python/)). I would use update or a dict comprehension.

Reply:
And [some people does not like it](https://mail.python.org/pipermail/python-dev/2010-April/099459.html) :)

Reply:
Yes, I wasn't aware that was so much debate online about the right way to merge dicts in Python (see https://stackoverflow.com/a/26853961/1791279). I did like this version but since it's controversial, reverted it to a simpler one. As a side note, as args expansion related hacks go you might like [this one](http://locallyoptimal.com/blog/2013/01/20/elegant-n-gram-generation-in-python/) which I have seen used in practice )